### ISSUE TITLE
Type error

### ISSUE THREAD

**Issue by IamSAB** [OleVik/grav-plugin-readability#3]
Installing the plugin breaks grav due to the following error:

```
 Whoops \ Exception \ ErrorException (E_COMPILE_WARNING)
"boolean" will be interpreted as a class name. Did you mean "bool"? Write "\Grav\Plugin\ReadabilityPlugin\boolean" or import the class with "use" to suppress this warning
```
I think you just should change the `boolean` type to `bool`:

```php
class FindResourceTwigExtension extends \Twig_Extension
{
    public function getName()
    {
        return 'FindResourceTwigExtension';
    }
    public function getFunctions()
    {
        return [
            new \Twig_SimpleFunction('findresource', [$this, 'getter'])
        ];
    }
    public function getter(string $uri, boolean $absolute = null, boolean $first = null)
```
in `/user/plugins/readability/twig/FindResourceExtension.php`

**Comment by Binero** [comment]
Can confirm this issue exists.

**Comment by Hazcrew** [comment]
I can confirm this issue still exists which is sad as its such an easy fix.
Just replace the 2 instances of 'boolean' with 'bool'
Note - a bit more info.   
it only seems to happen in PHP 8.0 and above. The code  was written for PHP 7 where 'boolean' is an alias for 'bool'.
However, boolean has been deprecated in PHP 8.

**Comment by OleVik** [comment]
Sorry, GitHub-triage took a while. Thanks for the succinct description of a fix, @Hazcrew, this has been released in [v2.1.1](https://github.com/OleVik/grav-plugin-readability/releases/tag/v2.1.1).
